lib/token-autocomplete.ts

```diff
@@ -446,6 +452,20 @@ class TokenAutocomplete {
             });
         }
 
+        handleInput(highlightedSuggestion: any): void {
```

Comment:
wieso gibt es die handleInput zwei mal?

Reply:
Einmal für den SingleSelect und einmal den MultiSelect Modus. Die Lib ist intern in zwei innere Klassen aufgeteilt um die Logik sauber zu trennen